### ISSUE TITLE
[Feature]: Hide Token Stack Indicator

### ISSUE THREAD

**Issue by Jmr3366** [RPTools/maptool#4326]
### Describe the Problem

There are times where I'd like an interactable token placed on the token layer so a player can "click" on it but also _not_ to show the token stack icon if a player moves over or stops on the _clickable token_.

This was something I'd looked for on discord a while back:
https://discord.com/channels/296230822262865920/1040271781115731979/1040271781115731979

### The Solution you'd like

An option to disable the token stack indicator

### Alternatives that you've considered.

Move tokens above FoW (that masks the problem known from another bug) and doesn't keep the click token _below_ the PC token
Move the token to another layer dynamically based off proximity of a PC token on the token layer
I can disable snap on tokens 

None of those would be prefered 

### Additional Context

![image](https://github.com/RPTools/maptool/assets/100969108/1f1b6455-0be2-4d8d-b765-a7c37022e07a)

**Comment by cwisniew** [comment]
If there is no token stack how does the player chose which to interact with?

**Comment by Jmr3366** [comment]
This is just the indicator.  Double-click behavior would remain the same.

**Comment by Baaaaaz** [comment]
Tried testing in 1.15.0 Beta 1 and could check the new 'Hide Token stack indicator' preference and set it by pressing the Okay button, however it did not take immediate affect on a visible set of stack tokens on the current map.  I also had to click on the map for the token indicator to disappear.

Similar behavior when I unchecked this new preference and pressed okay, the stack indicator only reappeared when I clicked on the map.

Should the stack indicators actually disappear/reappear once the OK button on the preferences dialog is clicked?
